src/Generation/GapicClientV2Generator.php

```diff
@@ -368,6 +371,39 @@ private function operationMethods(): Vector
             ));
         $methods = $methods->append($resumeOperation);
 
+        if ($this->serviceDetails->migrationMode === MigrationMode::NEW_SURFACE_ONLY) {
+            // write createOperationsClient method for new surface clients
+            $operationsClientType = $this->serviceDetails->hasCustomOp
+                ? $this->ctx->type($this->serviceDetails->customOperationServiceClientType)
+                : $this->ctx->type(Type::fromName(OperationsClient::class));
+            $createOperationsClient = AST::method('createOperationsClient')
+                ->withAccess(Access::PRIVATE)
+                ->withParams(AST::param(ResolvedType::array(), $options))
+                ->withBody(AST::block(
+                    AST::call(AST::method('unset'))(
+                        AST::index($options, 'serviceName'),
+                        AST::index($options, 'clientConfig'),
+                        AST::index($options, 'descriptorsConfigPath'),
+                    ),
```

Comment:
So why the `unset`? Maybe include a comment here including the "why".

Reply:
Yes I will do that!

To answer your question, it's a carry-over from [the createOperationsClient method](https://github.com/googleapis/gax-php/blob/710e47dddae8dd490cb02e6bb8f4cb7cadc77572/src/GapicClientTrait.php#L377-L381) in GAX which does the same thing. The reason is that those configs are CLIENT specific, so passing them down to the operations client will break it (it'll load the wrong configs/descriptors).

Reply:
Added the comment to the generated output